src/components/dashboard/PreviousTrips.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+/**
+ *  PreviousTrips displays a list of all previous
+ *  trips recorded by user
+ */
+
+export default class PreviousTrips extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hasPrevious: this.props.previous && this.props.previous.length
+    };
```

Comment:
If hasPrevious is never intended to change and subsequently be used, I think it would be better to write it into a helper method `hasPrevious()`. This would allow the functions `getAllPrevious` and `getPreviousDiv` to be unbound and the constructor to be removed.